src/libs/OptionsListUtils.ts

```diff
@@ -2494,7 +2494,7 @@ function getEmptyOptions(): Options {
 }
 
 function shouldUseBoldText(report: ReportUtils.OptionData): boolean {
-    return report.isUnread === true && report.notificationPreference !== CONST.REPORT.NOTIFICATION_PREFERENCE.MUTE;
+    return report.isUnread === true && ReportUtils.getReportNotificationPreference(report) !== CONST.REPORT.NOTIFICATION_PREFERENCE.MUTE;
```

Comment:
Missing the usage of the single source of truth i.e. `report.notificationPreference` resulted in #51410

Reply:
@puneetlath Well! I am not aware of the context behind deprecating `report.notificationPreference`. Let me know if I am missing something here. But, the reasoning for the decision is [here](https://github.com/Expensify/App/issues/51410#issuecomment-2456324017) i.e.

> The purpose of `optionData` was to have only as much report data as needed. So, adding `participants` does not seem a good idea especially since we have already captured `notificationPreference` as the single source of truth.